.gitignore

```diff
@@ -6,4 +6,6 @@ coverage
 .nyc_output
 /dist
 /stats
-/components
\ No newline at end of file
+/components
+/.prettierrc.json
```

Comment:
why was prettier used? We already have eslint set up and an eslintrc on the root.

Reply:
Please make sure code is formatted using eslint and the config already present.

Reply:
321bd4ced83925829ae7a889f3d16964b6fa0c12